docs/app/Examples/modules/Dropdown/Types/DropdownExampleInlineTwo.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { Dropdown, Header, Icon } from 'semantic-ui-react'
+
+/**
+ * Heads up!
+ *
+ * SUI core has these 4 fields on the Dropdown.Item:
+ *
+ * "name"  : "Choice 1", // name displayed in dropdown
+ * "value" : "value1", // selected value
+ * "text"  : "Choice 1" // name displayed after selection (optional)
+ * "disabled"  : false // whether field should be displayed as disabled (optional)
+ *
+ * We don't currently make a distinction between 'text' and 'name', but this
+ * example does in the core docs. It would be a breaking change and so let's
```

Comment:
It's scattered, but the main sentiment is captured here: http://semantic-ui.com/modules/dropdown.html#specifying-different-text--hidden-input-values

You can see an example in the SUI core docs in the second example here: http://semantic-ui.com/modules/dropdown.html#inline. Notice how the text shown is lowercased but the text in the dropdown is title-cased.

Lastly look at the `Expected server response` here: http://semantic-ui.com/modules/dropdown.html#remote-content. Pasting the relevant snippet here:

```
{
  // name displayed in dropdown
  "name"  : "Choice 1",
   // selected value
  "value" : "value1",
   // name displayed after selection (optional)
  "text"  : "Choice 1"
   // whether field should be displayed as disabled (optional)
  "disabled"  : false
},
```